pyresample/test/test_ewa_ll2cr.py

```diff
@@ -66,9 +77,33 @@ def test_lcc_basic1(self):
         w = grid_info["width"]
         h = grid_info["height"]
         points_in_grid = _ll2cr.ll2cr_static(lon_arr, lat_arr, fill_in, proj_str,
-                                                               cw, ch, w, h, ox, oy)
+                                             cw, ch, w, h, ox, oy)
         self.assertEqual(points_in_grid, lon_arr.size, "all these test points should fall in this grid")
 
+    def test_geo_antimeridian(self):
+        """ Ensure output for anti-meridian crossing input includes all points. """
+        from pyresample.ewa import _ll2cr
+        lon_arr = create_test_longitude(160.0, 200.0, (50, 100), dtype=np.float64)
+        lat_arr = create_test_latitude(40.0, 60.0, (50, 100), dtype=np.float64)
+
+        # wrap values in lon_arr so -180 ≤ longitude (degrees east) ≤ 180
+        lon_arr[lon_arr > 180] -= 360.0
+        print(lon_arr[0])
```

Comment:
Left over print statement. Could you remove this?

Reply:
D'oh! I've removed that now. Thanks.